flash-messages/app-flash-messages.rb

```diff
@@ -0,0 +1,44 @@
+require "sinatra"
+
+# We point DataMapper to the correct database in setup.rb
+require "./setup"
+
+# models.rb is where we define our Chirp model
+require "./models"
+
+# To use flash messages, we need to require the Flash middleware
+require "rack-flash"
+
+# Because flash middleware uses sessions, we need to enable sessions
+set(:sessions, true)
+
+# Make sure to use a session secret!
+set(:session_secret, ENV["SESSION_SECRET"])
+
+# Finally, to enable the `flash` method we add Rack::Flash to our middleware
+use Rack::Flash
```

Comment:
Oh! So this is a set of 3 grouped things. We may want to group them in a method. `enable_rack_flash_middleware` or somthing.